hack/bundle-automation/config.yaml

```diff
@@ -9,7 +9,7 @@
 
 - repo_name: image-based-install-operator
   github_ref: "https://github.com/openshift/image-based-install-operator.git"
-  branch: "backplane-2.5"
+  branch: "main"
```

Comment:
Using `main` branch as reference until `backplane-2.6` branch is created for `image-based-install-operator`.